x/rewards/spec/README.md

```diff
@@ -0,0 +1,83 @@
+<!--
+order: 0
+title: Rewards Overview
+parent:
+  title: "rewards"
+-->
+
+# `Rewards`
+
+## Overview
```

Comment:
🤔 this might be nitpicking but the Cosmos SDK spec modules use Abstract for this title rather than Overview